robot/Cumulus/resources/RecurringDonationsPageObject.py

```diff
@@ -198,6 +209,35 @@ def get_next_payment_date_number(self, paynum):
         newString = f"{dd}/{mm}/{tokens[2]}"
         return newString
 
+    @capture_screenshot_on_error
+    def validate_current_and_next_year_values(self, amount):
+        """Takes in the parameter current installment payment (amount)
+        calculates the current and next year value payments and
+        validates them with the values displayed on the UI. """
+        installmentrow = npsp_lex_locators["erd"]["installment_row"]
+        installments = self.selenium.get_webelements(installmentrow)
+        count = len(installments)
+        print(f"Number of installments created is {count}")
+        i = 1
+        curr_year_value = 0
+        next_year_value = 0
+        values = {}
+        while i < count:
```

Comment:
what if count is 0 ? Should you be raising an exception for it ?

Reply:
Thanks Safiya. Good point and added exception